### PR TITLE
Add forecasted_write_load and forecasted_shard_size_in_bytes to the endpoint

### PR DESCRIPTION
**Pull request by idegtiarenko**

### CODE REVIEW

File: server/src/main/java/org/elasticsearch/action/admin/cluster/allocation/DesiredBalanceResponse.java [hunk 1]
```diff
@@ -165,7 +187,12 @@ public void writeTo(StreamOutput out) throws IOException {
             out.writeBoolean(relocatingNodeIsDesired);
             out.writeVInt(shardId);
             out.writeString(index);
-            out.writeOptionalWriteable(allocationId);
+            if (out.getVersion().onOrAfter(ADD_FORECASTS_VERSION)) {
+                out.writeOptionalDouble(forecastedWriteLoad);
+                out.writeOptionalLong(forecastedShardSizeInBytes);
+            } else {
+                out.writeMissingWriteable(AllocationId.class);
```
Comment (fcofdez) [inline server/src/main/java/org/elasticsearch/action/admin/cluster/allocation/DesiredBalanceResponse.java:194]: Why are we removing the allocation id?

**Review by fcofdez — COMMENTED** [review]
This looks good to me, I left a question though

File: server/src/main/java/org/elasticsearch/action/admin/cluster/allocation/DesiredBalanceResponse.java (hunk 1 above)
Comment (idegtiarenko) [reply server/src/main/java/org/elasticsearch/action/admin/cluster/allocation/DesiredBalanceResponse.java:194]: Only to remove unused field

**Review by fcofdez — APPROVED** [review]
LGTM 👍

**Pull request merged**